README.md

```diff
@@ -18,427 +18,15 @@ A modern, lightweight learning management system.
-$ sudo -u postgres createuser --superuser $USER
-$ sudo -u postgres createdb $USER
-$ psql
-```
-
-Create the Doubfire user the following at the Postgres prompt:
-
-```
-CREATE ROLE itig WITH CREATEDB PASSWORD 'd872$dh' LOGIN;
-```
-
-#### 4. Install native tools
-
-Install `imagemagick`, `libmagic` and `ghostscript`. You will also need to
-install the Python `pygments` package:
-
-```
-$ sudo apt-get install ghostscript \
-                       imagemagick \
-                       libmagickwand-dev \
-                       libmagic-dev \
-                       python-pygments
-```
-
-#### 5. Install Doubtfire API dependencies
-
-Clone project and change your working directory to the api:
-
-```
-$ git clone https://github.com/doubtfire-lms/doubtfire-api.git
-$ cd ./doubtfire-api
-```
-
-Comply with [DSTIL](https://github.com/dstil) best practices using DSTIL's [dotfiles](https://github.com/dstil/dotfiles) repo. Run the following command to install DSTIL's dotfiles.
-
-```
-$ curl -s https://raw.githubusercontent.com/dstil/dotfiles/master/bootstrap | bash
-```
-
-Restart your terminal so that you can run `dstil`:
-
-```
-$ exec $SHELL
-$ which dstil
-dstil: aliased to /Users/Fred/.dstil/current/sources/1/files/labcheck
-```
-
-Set up [overcommit](https://github.com/brigade/overcommit) and install hooks:
-
-```
-$ gem install overcommit
-$ rbenv rehash
-$ overcommit --install
-$ dstil --sign
-```
-
-Then install Doubtfire API dependencies using [bundler](http://bundler.io):
-
-```
-$ gem install bundler
-$ rbenv rehash
-$ bundle install --without production
-```
-
-##### Bundle resolutions
-
-You may encounter issues when trying to install bundle dependencies.
-
-###### ruby-filemagic
-
-The `ruby-filemagic` gem cannot find `libmagic` libraries when compiling with native extensions:
-
-```
-Installing ruby-filemagic 0.6.0 with native extensions
-
-Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.
-
-    /Users/[User]/.rbenv/versions/2.3.1/bin/ruby extconf.rb
-checking for magic_open() in -lmagic... no
-checking for magic.h... no
-*** ERROR: missing required library to compile this module
-*** extconf.rb failed ***
-Could not create Makefile due to some reason, probably lack of necessary
-libraries and/or headers.  Check the mkmf.log file for more details.  You may
-need configuration options.
-```
-
-To resolve, add the following to your global bundle config:
-
-```
-$ bundle config build.ruby-filemagic --with-magic-include=/usr/local/include --with-magic-lib=/usr/lib
-```
-
-Then try installing dependencies again.
-
-#### 6. Create and populate Doubtfire
-
-##### Environment variables
+#### 2. Manual install
+The manual installation guide can be found on the wiki for: [Linux](https://github.com/doubtfire-lms/doubtfire-api/wiki/Linux-setup), [MacOS](https://github.com/doubtfire-lms/doubtfire-api/wiki/MacOS-setup), or [Docker](https://github.com/doubtfire-lms/doubtfire-api/wiki/Docker-setup)
```

Comment:
Missing period for sentence. Lowercase m for macOS. Should `{Linux|macOS|Docker}-Setup` have a capital S?